lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -218,6 +233,18 @@ def cmd_workspace(*args)
     end
   end
 
+  def print_deleted_workspaces(deleted_workspaces, starting_ws)
+    deleted_workspaces.each do |ws|
+      if ws.name == Msf::DBManager::Workspace::DEFAULT_WORKSPACE_NAME
+        print_status 'Deleted and recreated the default workspace'
```

Comment:
Is it possible to delete and recreate the default workspace and also switch to it?